helm/resource_release.go

```diff
@@ -1508,3 +1513,16 @@ func valuesKnown(d *schema.ResourceDiff) bool {
 	}
 	return true
 }
+
+func useChartVersion(chart string, repo string) bool {
+	_, urlerr := url.ParseRequestURI(chart)
```

Comment:
I think you probably want to be more specific in this check as this will return true when setting the chart to a chart in a OCI registry. 

It seems like you only want this to return true if chart is a URL to a helm chart archive.

Reply:
this was a good catch, I covered this with the following check

```go
func useChartVersion(chart string, repo string) bool {
	// checks if chart is a URL or OCI registry
	if _, urlerr := url.ParseRequestURI(chart); urlerr == nil && !registry.IsOCI(chart) {
		return true
	}
```

so it will check if the `chart` is a URL and also will check if chart is not an OCI registry